CONTRIBUTING.md

```diff
@@ -92,26 +92,28 @@ If your commit messages do not contain enough information for a reviewer to unde
  3. Contributions to the schema that break backwards compatibility
 
-Each of these comes with varying requirements for review and testing. While core contributors *can* commit directly to the develop branch, they should reserve this ability for all but the most trivial of changes (e.g., typos).
+Each of these comes with varying requirements for review and testing. While Core Contributors *can* commit directly to the develop branch, they should reserve this ability for all but the most trivial of changes (e.g., typos).
 
-For *all other types of contributions*, core contributors should file a pull request containing their proposed changes. This pull request will be reviewed by at least one other core contributor for inclusion. Merging the contribution is the responsibility of the reviewing contributor.
+For *all other types of contributions*, Core Contributors should file a pull request containing their proposed changes. This pull request will be reviewed by at least one other core contributor for inclusion. Merging the contribution is the responsibility of the reviewing contributor.
 
-Above all, however, core contributors are asked to exercise wisdom and common sense in merging contributions. If they are unsure of the impact of a proposed change, or the documentation of the contribution (pull request messages, commit messages, discussions) are not answered to their satisfaction, they should err on the side of caution and not merge until they are completely satisfied, or call in help from other members of the community.
+Above all, however, Core Contributors are asked to exercise wisdom and common sense in merging contributions. If they are unsure of the impact of a proposed change, or the documentation of the contribution (pull request messages, commit messages, discussions) are not answered to their satisfaction, they should err on the side of caution and not merge until they are completely satisfied, or call in help from other members of the community.
 
 ### Working with Pull Requests Locally
 
 Sometimes it's easiest if you can check out a pull request's changes locally and run it on your own machine. To do this, you can follow [these instructions](https://gist.github.com/piscisaureus/3342247). This will allow you to further test proposed modifications.
 
-### TravisCI
+### Github Actions
 
-When a new pull request comes in, the TravisCI service will automatically build the schema using the TEIC Stylesheets, and then validate the test files and the sample encodings against the built schemas. If, for some reason, a change breaks the schema build, or breaks validation, TravisCI will report an error on the pull request. It is then up to the original contributor to fix the issue and submit the fix to the pull request.
+When there is a new push to the develop branch, a GitHub Actions workflow will automatically build the MEI schema files and guidelines and publish them at: https://github.com/music-encoding/schema/tree/main/dev
```

Comment:
Sorry,  missed this before: Is it correct that the guidelines are published at the schema repo? Should we better mention the guidelines repo, too?

Reply:
thx for spotting missed that on :-o